packages/plugins/plugin-deck/src/components/DeckLayout/ComplementarySidebar.tsx

```diff
@@ -18,41 +19,45 @@ import { useNode, useNodeActionExpander } from '../../hooks';
 import { useLayout } from '../LayoutContext';
 
 export type ComplementarySidebarProps = {
-  id?: string;
+  context?: string;
   layoutParts: LayoutParts;
   flatDeck?: boolean;
 };
 
-export const ComplementarySidebar = ({ id, layoutParts, flatDeck }: ComplementarySidebarProps) => {
+export const ComplementarySidebar = ({ context, layoutParts, flatDeck }: ComplementarySidebarProps) => {
   const { popoverAnchorId } = useLayout();
+  const attentionPlugin = useResolvePlugin(parseAttentionPlugin);
+  const attention = attentionPlugin?.provides.attention;
+  const attended = attention?.attended ? Array.from(attention.attended) : [];
+  const id = attended[0] ? `${attended[0]}${SLUG_PATH_SEPARATOR}${context}` : undefined;
   const { graph } = useGraph();
   const node = useNode(graph, id);
-  // const complementaryAvailable = useMemo(() => id === NAV_ID || !!node, [id, node]);
   const complementaryAttrs = createAttendableAttributes(id?.split(SLUG_PATH_SEPARATOR)[0] ?? 'never');
 
   useNodeActionExpander(node);
 
   return (
     <Main.ComplementarySidebar {...complementaryAttrs}>
-      {node ? (
-        <div role='none' className={mx(deckGrid, 'grid-cols-1 bs-full')}>
-          <NodePlankHeading
-            node={node}
-            id={id}
-            layoutParts={layoutParts}
-            layoutPart='complementary'
-            popoverAnchorId={popoverAnchorId}
-            flatDeck={flatDeck}
-          />
+      <div role='none' className={mx(deckGrid, 'grid-cols-1 bs-full')}>
```

Comment:
`deckGrid` in the sidebar? Is it for the rows?

Reply:
question back at you...not sure, this is just what was here 😅

Reply:
Oh, derp, right, this was just a change in the ternary expression. Nevermind 😅